src/components/ClassroomComponent/MyEventsComponent/MyEventsComponent.js

```diff
@@ -276,6 +293,7 @@ class MyEventsComponent extends Component {
                           <MenuItem primaryText='View' onClick={() => this.showDialog(event)} />
                           <MenuItem hidden={!event.receiptURL} primaryText='Post Event Details' onClick={() => this.showPostEventDetailDialog(event)} />
                           <MenuItem hidden={!event.receiptURL} primaryText='Download Receipt' onClick={() => { window.location = (event.receiptURL) }} />
+                          <MenuItem hidden={event.startDate < today} primaryText='Cancel Event' onClick={() => this.showCancelEventDialog(event)} />
```

Comment:
Are you sure string comparison will work in this case? I dont think you can compare 2 dates like so. Use `moment.js` for this. Checkout the [momentjs](https://momentjs.com/) docs